internal/engine/wazevo/backend/isa/arm64/instr.go

```diff
@@ -210,12 +214,16 @@ var useKinds = [numInstructionKinds]useKind{
 	fpuCSel:              useKindRNRM,
 	movToVec:             useKindRN,
 	movFromVec:           useKindRN,
+	movFromVecSigned:     useKindRN,
 	vecDup:               useKindRN,
 	vecExtract:           useKindRNRM,
 	cCmpImm:              useKindRN,
 	vecMisc:              useKindRN,
+	vecMovElement:        useKindRN,
 	vecLanes:             useKindRN,
 	vecShiftImm:          useKindRN,
+	vecTbl:               useKindRNRM,
+	vecTbl2:              useKindRNRM,
```

Comment:
FYI: in order for the regalloc to work as expected `vecTbl2` must indicate that these consecutive regs in our case v29 and v30.

Reply:
something like this?

```
	case useKindRNRN1RM:
		if rn := i.rn.reg(); rn.Valid() {
			regs = append(regs, rn)
		}
		if rn1 := i.rn.reg() + 1; rn1.Valid() {
			regs = append(regs, rn1)
		}
		if rm := i.rm.reg(); rm.Valid() {
			regs = append(regs, rm)
		}
```

Reply:
probably too simplistic (`panic: BUG color conflict: r61 vs r62`)

Reply:
maybe...

```
	case useKindRNRN1RM:
		if rn := i.rn.reg(); rn.Valid() {
			regs = append(regs, rn)
		}
		if rm := i.rm.reg(); rm.Valid() {
			regs = append(regs, rm)
		}
```

and then...

```
		if index == 0 {
			if rn := i.rn.reg(); rn.Valid() {
				i.rn = i.rn.assignReg(reg)
			}
			if rn1 := i.rn.reg() + 1; rn1.Valid() {
				i.rm = i.rm.assignReg(reg + 1)
			}
		} else {
			if rm := i.rm.reg(); rm.Valid() {
				i.rm = i.rm.assignReg(reg)
			}
		}
```

Reply:
(thought I found a solution and LOL it's passing _again_ for pure luck because it was really wrong :D overwriting rm twice)